extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -123,9 +123,9 @@ public function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 	 * @throws \MWException
 	 */
 	public function onSkinAfterBottomScripts( $skin, &$bottomScripts ) {
-		global $wgUseSiteJs;
+		global $wgOut, $wgUseSiteJs;
 
-		if ( !empty( $wgUseSiteJs ) ) {
+		if ( !empty( $wgUseSiteJs ) && $wgOut->isUserJsAllowed() ) {
```

Comment:
Here we can use the `RequestContext` provided by the `Skin` instance passed to the hook handler instead of relying on global state: `$skin->getOutput()`